ansible/group_vars/all.yml

```diff
@@ -31,10 +24,12 @@ cni: weave
 # Configure Kubernetes container runtime (CRI)
 # Options: "containerd" or "docker"
 # Default: "containerd"
-cri: containerd
+cri: docker
+
+# Role: common
+common_timezone: Europe/Amsterdam
+common_rpi_overclock: true
+common_rpi_poe_hat: true
 
-# Network interface
-# Note: Flash SD with wlan0 configuration doesn't guarantee IP allocation is static and match as defined in eth0
-# Options: "eth0" or "wlan0"
-# Default: "eth0"
-network_interface: eth0
+# Role: cluster
+cluster_control_plane_endpoint: api.kubernetes.lan:8443
```

Comment:
What is the kubernetes default and should we consider using it as the default?

Reply:
There is no k8s default. Currently set the default to be `keepalive_vip`.